walkthroughSteps/step-4/src/dynamic.js

```diff
@@ -1,8 +1,7 @@
 const dbConnection = require('../database/db_connection.js');
 
-const getData = (cb) =>{
-    dbConnection.query(`SELECT * FROM superheroes`
-    , (err, res) => {
+const getData = cb => {
+  dbConnection.query(`SELECT * FROM superheroes`, (err, res) => {
     if (err) cb(err);
```

Comment:
This either needs to be an `if/else` or there needs to be an early return in the `if` statement, otherwise an error will cause the callback to execute twice. Let's double check the rest of the steps too to see if it crops up in any of them too.